### PR TITLE
Update UI Documentation

### PR DESCRIPTION
**Pull request by ben-polinsky**
Incorporates documentation changes made by @GerardasB and the AppUI team. Links targeting the old AppUI docs have been updated to point to the new docs, or an appropriate API.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -112,10 +112,11 @@ rush.json       @iTwin/itwinjs-core-admins
 /core/webgl-compatibility       @iTwin/itwinjs-core-display @Ellord207
 
 /docs   @iTwin/itwinjs-core
-/docs/getting-started @iTwin/itwinjs-core-admins @williamkbentley
-/docs/config          @iTwin/itwinjs-core-admins @williamkbentley
+/docs/getting-started @iTwin/itwinjs-core-admins @anmolshres98 @hl662 @DanRod1999
```
Comment (DanRod1999) [inline .github/CODEOWNERS:115]: Mike B is still mentioned in this file and it seems to be throwing an error. Can we fix int this PR?

**Review by aruniverse — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (ben-polinsky) [reply .github/CODEOWNERS:115]: Yep

Comment (ben-polinsky) [reply .github/CODEOWNERS:115]: Fixed

**Comment by GerardasB** [comment]
> In addition, is there a timeframe for those alpha APIs you planned on promoting?

Fixed. 
We should get to https://github.com/iTwin/appui/issues/823 late next week.
Do we have to merge this? I thought we were moving the docs to `appui` repository.

**Review by GerardasB — APPROVED** [review]

**Comment by ben-polinsky** [comment]
> I thought we were moving the docs to `appui` repository.

I am not sure when I'll have the time to get changes in to support this. Might be this week, but might have to deprioritize.

**Comment by ben-polinsky** [comment]
@mergifyio backport release/4.7.x

**Pull request merged**